models/gc_spider_baseline/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM mhubai/base:latest
+
+# Update authors label
+LABEL authors="sil.vandeleemput@radboudumc.nl"
+
+# Install PyTorch 2.0.1 (CUDA enabled)
+RUN pip3 install --no-cache-dir torch==2.0.1+cu118 -f https://download.pytorch.org/whl/torch_stable.html
+
+# Install git-lfs (required for unpacking model weights)
+RUN apt update && apt install -y --no-install-recommends git-lfs && rm -rf /var/lib/apt/lists/*
+
+# Install spider baseline algorithm
+#  - Git shallow clone to tmp directory
+#  - Extract relevant files to src
+#  - Remove tmp directory
+RUN git clone --depth 1 https://github.com/DIAGNijmegen/SPIDER-Baseline-IIS.git /tmp/algorithm && \
+    mkdir -p src/datasets/spider_input/images && \
+    mv /tmp/algorithm/devel src/devel && \
+    mv /tmp/algorithm/experiments src/experiments && \
+    rm -rf /tmp/algorithm
+
+# Install additional spider baseline dependencies
+RUN pip3 install --no-cache-dir git+https://github.com/DIAGNijmegen/Tiger.git@stable
+
+# TODO Clone the main branch of MHubAI/models
+#RUN git stash \
+#    && git fetch https://github.com/MHubAI/models.git main \
+#    && git merge FETCH_HEAD \
+#    && git sparse-checkout set "models/gc_spider_baseline" \
+#    && git fetch https://github.com/MHubAI/models.git main
+
+# Add two static configuration json files required by the SPIDER baseline algorithm
+RUN echo '{"filters": 50, "traversal_direction": "up"}' > /app/src/experiments/SPIDER-Baseline/arguments.json
```

Comment:
> Hey Sil, would there be any use case where one want to derive from these static parameters?
> 
maybe the `traversal_direction`, filters should remain 50 to match the model

> If yes, I'd suggest we expose (some of) them via @IO.Config and construct the json at Module initialization.
> 
fine, the `traversal_direction` can be exposed.

> In general, could we move these as files into `/app/models/$model_name/src`? The current implementation is totally valid; however it won't scale well for large configs.

It should be possible to do so. I will not do this for the current algorithms, because it is quite some work and falls beyond the scope of the two months earlier established quality checklist. If you want to do so feel free to do it.

Reply:
>  If you want to do so feel free to do it.

??

Reply:
These options have been added now.